### PR TITLE
2.3.2-beta2 release

### LINKED ISSUE #8846: App crashes on opening external file/cmd-clicking on it
When attempting to open a file in an external editor, the app crashes. This is very inconsistent. Sometimes it's when opening in an external editor, sometimes it's just when command-clicking on a modified file. It is occurring frequently. The steps below are one example of when it occurred, although I could then not reproduce it using the same steps.

This frequent crashing started after updating to 2.2.4.

GitHub Desktop version 2.2.4
MacOS version 10.15.2

1. Visual Studio Code set as the external editor in preferences
2. Make a change in a Podfile.lock file
3. Command-click on the file in the list and choose Open with Visual Studio Code 
4. Application crashes

### LINKED ISSUE #9078: Exits when using 'discard changes'
### Describe the bug

When attempting to 'Discard Changes' to a modified file, GitHub Desktop exits with an exception report

### Version & OS

GitHub Desktop: Version 2.3.1
OS: Catalina Version 10.15.3

### Steps to reproduce the behavior

1. Right mouse click on modified file in Changes list
2. Select "Discard Changes..."
3. Click 'Discard Changes' button

### Expected behavior

Changes discarded and GitHub Desktop is ready for next action
### Actual behavior

Github Desktop disappears and a few seconds later the log monitor pops up asking to send the log to Apple. 

### Screenshots

No screenshot

### Logs

[2020-02-09.desktop.production.log](https://github.com/desktop/desktop/files/4179856/2020-02-09.desktop.production.log)

### Additional context

No other comment.

### PR DESCRIPTION
**Pull request by outofambit**
- [x] finalize release notes
- [x] release!

Fixes from upgrade to Electron 7:
Closes #8846 
Closes #9078

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.3.2-beta2": [
+      "[Fixed] Text in error dialogs is now selectable - #7606",
+      "[Fixed] Prevent some stash and move operations from failing silently - #9038",
+      "[Fixed] Removing branches succeeds even if branch is already deleted on remote - #8831",
+      "[Improved] Errors containing raw Git output are now formatted using fixed-width font - #8964",
+      "[Improved] Upgrade to Electron 7 - #8967"
```
Comment (outofambit) [inline changelog.json]: i know we don't usually note these things, but it seems helpful from a record-keeping debugging standpoint?

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.3.2-beta2": [
+      "[Fixed] Text in error dialogs is now selectable - #7606",
```
Comment (outofambit) [inline changelog.json]:
```suggestion
      "[Fixed] Make text in error dialogs selectable - #7606",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+      "[Fixed] Prevent some stash and move operations from failing silently - #9038",
+      "[Fixed] Removing branches succeeds even if branch is already deleted on remote - #8831",
```
Comment (outofambit) [inline changelog.json]:
```suggestion
      "[Fixed] Ignore some remote errors when deleting a local branch - #8831",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,12 @@
...
+      "[Improved] Errors containing raw Git output are now formatted using fixed-width font - #8964",
```
Comment (outofambit) [inline changelog.json]:
```suggestion
      "[Improved] Format errors containing raw Git output with a fixed-width font - #8964",
```

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.3.2-beta2": [
+      "[Fixed] Text in error dialogs is now selectable - #7606",
+      "[Fixed] Prevent some stash and move operations from failing silently - #9038",
```
Comment (outofambit) [inline changelog.json]:
```suggestion
      "[Fixed] Prevent bringing changes to another branch from failing silently - #9038",
```

File: changelog.json (hunk 1 above)
Comment (billygriffin) [reply changelog.json]: I don't want to put it in the prod release notes, but if you think there's value in including in the beta ones, I'll leave that up to you to make the call on.

File: changelog.json [hunk 6]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.3.2-beta2": [
+      "[Fixed] Make text in error dialogs selectable - #7606",
+      "[Fixed] Prevent bringing changes to another branch from failing silently - #9038",
+      "[Fixed] Ignore some remote errors when deleting a local branch - #8831",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Ensure local branch is also deleted when deleting a branch that no longer exists on the remote - #8831",
```

File: changelog.json [hunk 7]
```diff
@@ -1,5 +1,12 @@
...
+      "[Improved] Format errors containing raw Git output with a fixed-width font - #8964",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Format errors containing raw Git output with fixed-width font - #8964",
```

File: changelog.json (hunk 1 above)
Comment (outofambit) [reply changelog.json]: yeah definitely not in prod!

**Comment by outofambit** [comment]
The [mac release build failed](https://app.circleci.com/jobs/github/desktop/desktop/12203) (disregard the approved status shown in the merge box, that is wrong) due to https://github.com/desktop/dugite-native/issues/318.

**Comment by outofambit** [comment]
closing this until we're ready to revive it with the fix from desktop/dugite-native#319

**Pull request merged**